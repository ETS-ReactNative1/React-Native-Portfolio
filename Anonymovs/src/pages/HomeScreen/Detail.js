import React, {useState} from 'react';
import {
  ImageBackground,
  Image,
  StyleSheet,
  Text,
  View,
  ScrollView,
  TouchableOpacity,
} from 'react-native';
import Colors from '../../../assets/config/Colors';
import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
import {useDispatch, useSelector} from 'react-redux';
import {
  widthPercentageToDP as wp,
  heightPercentageToDP as hp,
} from 'react-native-responsive-screen';
import LinearGradient from 'react-native-linear-gradient';
import YoutubePlayer from 'react-native-youtube-iframe';
import {Rating, AirbnbRating} from 'react-native-ratings';
import {actionRatingSubmit, addFavorite, removeFavorite} from '../../redux/action/actions';

export default function Detail({navigation}) {
  const dispatch = useDispatch();
  const {detail, video, review, rating, favorites} = useSelector(
    state => state.allReducers,
  );
  const [playing, setPlaying] = useState(false);
  const [postRating, setPostRating] = useState(5);

  const addToFavorites = movie => dispatch(addFavorite(movie));
  const removeFromFavorites = movie => dispatch(removeFavorite(movie));
  const handleAddFavorite = movie => {
    addToFavorites(movie);
  };
  const handleRemoveFavorite = movie => {
    removeFromFavorites(movie);
  };

  const exists = movie => {
    if (favorites?.filter(detail => detail.id === movie.id).length > 0) {
      return true;
    }
    return false;
  };

  return (
    <ScrollView showsVerticalScrollIndicator={false} style={styles.container}>
      <ImageBackground
        style={styles.backdrop}
        source={{
          uri: 'https://image.tmdb.org/t/p/w500' + detail.backdrop_path,
        }}>
        <LinearGradient
          start={{x: 0, y: 0}}
          end={{x: 0, y: 1}}
          colors={['transparent', Colors.background]}
          style={{
            width: '100%',
            height: hp(55),
            alignItems: 'center',
            justifyContent: 'flex-end',
          }}></LinearGradient>
        <View style={styles.posterShadow}>
          <Image
            resizeMode="contain"
            style={styles.poster}
            source={{
              uri: 'https://image.tmdb.org/t/p/w500' + detail.poster_path,
            }}
          />
        </View>
      </ImageBackground>
      <View style={styles.details}>
        <View style={styles.titleWrap}>
          <Text style={styles.title}>{detail.title}</Text>
          <TouchableOpacity
            onPress={() =>
              exists(detail)
                ? handleRemoveFavorite(detail)
                : handleAddFavorite(detail)
            }>
            <Icon name={exists(detail) ? "heart" : "heart-outline"} size={hp(3.5)} color={Colors.active} />
          </TouchableOpacity>
        </View>
        <View style={styles.ratingWrap}>
          <Icon name="star" size={20} color="gold" />
          <Text style={styles.rating}>
            {(detail.vote_average + rating) / 2}
          </Text>
        </View>
        <Text style={styles.detailTxt}>{detail.overview}</Text>
        <View style={styles.video}>
          <YoutubePlayer height={220} play={playing} videoId={video[0]?.key} />
        </View>
        <View style={styles.postRating}>
          <Text style={styles.rateMovieTxt}>Rate this Movie</Text>
          <AirbnbRating
            count={10}
            reviews={[
              'Terrible',
              'Bad',
              'Meh',
              'Not So Bad',
              'So So',
              'Good',
              'Very Good',
              'Wow',
              'Amazing',
              'Unbelievable',
            ]}
            defaultRating={5}
            size={20}
            onFinishRating={setPostRating}
          />
          <TouchableOpacity
            onPress={() => {
              dispatch(actionRatingSubmit(postRating));
              alert('Submit Success !');
            }}
            style={styles.submitBtn}>
            <Text style={styles.submitTxt}>Submit</Text>
          </TouchableOpacity>
        </View>
        <View>
          {review.map(item => {
            console.log('itm', item);
            return (
              <View key={item.id}>
                <View style={styles.author}>
                  <Text style={styles.authorName}>{item.author}</Text>
                  <Text style={styles.authorDate}>
                    {item.updated_at?.slice(0, 10)}
                  </Text>
                </View>
                <ScrollView style={styles.contentWrap}>
                  <Text style={styles.authorContent}>{item.content}</Text>
                </ScrollView>
              </View>
            );
          })}
        </View>
      </View>
    </ScrollView>
  );
}

const styles = StyleSheet.create({
  container: {
    backgroundColor: Colors.background,
    flex: 1,
  },
  backdrop: {
    width: wp(100),
    resizeMode: 'cover',
  },
  poster: {
    width: wp(40),
    height: hp(30),
    position: 'absolute',
    bottom: hp(10),
    left: wp(5),
  },
  posterShadow: {
    shadowColor: '#fff',
    shadowOffset: {
      width: 0,
      height: 0,
    },
    shadowOpacity: 0.39,
    shadowRadius: 8.3,
    elevation: 20,
  },
  details: {
    top: -hp(7),
    marginHorizontal: wp(5),
  },
  titleWrap: {
    flexDirection: 'row',
    justifyContent: 'space-between',
    alignItems: 'center',
  },
  title: {
    fontSize: 19,
    color: Colors.white,
    fontWeight: '600',
  },
  ratingWrap: {
    flexDirection: 'row',
    alignItems: 'center',
  },
  rating: {
    color: Colors.white,
    marginLeft: 10,
  },
  detailTxt: {
    color: Colors.ash,
    textAlign: 'justify',
    marginTop: hp(2),
  },
  video: {
    marginVertical: hp(2),
  },
  author: {
    flexDirection: 'row',
    justifyContent: 'space-between',
    alignItems: 'center',
    flex: 1,
  },
  authorName: {
    color: Colors.white,
    fontSize: 16,
  },
  authorDate: {
    color: Colors.white,
  },
  authorContent: {
    color: Colors.ash,
  },
  contentWrap: {
    height: hp(20),
    marginTop: hp(1),
    marginBottom: hp(3),
  },
  rateMovieTxt: {
    color: Colors.white,
    fontWeight: '500',
    textAlign: 'center',
  },
  postRating: {
    marginBottom: hp(3),
  },
  submitBtn: {
    alignItems: 'center',
    justifyContent: 'center',
    backgroundColor: Colors.active,
    borderRadius: wp(2),
    marginTop: hp(2),
  },
  submitBtnDone: {
    alignItems: 'center',
    justifyContent: 'center',
    backgroundColor: Colors.ash,
    borderRadius: wp(2),
    marginTop: hp(2),
  },
  submitTxt: {
    color: Colors.white,
    paddingVertical: hp(1),
  },
});
