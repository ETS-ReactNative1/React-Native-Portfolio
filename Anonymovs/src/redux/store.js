import {createStore, applyMiddleware} from 'redux';
import createSagaMiddleware from 'redux-saga';
import sagas from './saga/sagas';
import combineReducers from './reducer/combine'

const sagaMiddleware = createSagaMiddleware();
const store = createStore(combineReducers,applyMiddleware(sagaMiddleware));

sagaMiddleware.run(sagas);

export default store;