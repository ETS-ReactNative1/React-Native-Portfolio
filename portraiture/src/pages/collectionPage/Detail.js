import React, {useEffect, useState} from 'react';
import {
  ImageBackground,
  SafeAreaView,
  StatusBar,
  StyleSheet,
  Text,
  View,
  TouchableOpacity,
} from 'react-native';
import {useDispatch, useSelector} from 'react-redux';
import colors from '../../../assets/config/colors';
import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
import {
  actionGetAllCollection,
  actionGetOneCollection,
} from '../../redux/actions/action';
import Loading from '../../components/reusable/Loading';
import AsyncStorage from '@react-native-async-storage/async-storage';
import axios from 'axios';

export default function CollectionDetail(props) {
  const {oneCollection} = useSelector(state => state.collectionReducer);
  const dispatch = useDispatch();
  const [show, setShow] = useState(false);
  const [loading, setLoading] = useState(false);

  const item = props.route.params.item;

  const delCollection = async () => {
    const token = await AsyncStorage.getItem('token');
    var config = {
      method: 'delete',
      url:
        'https://portraiture.gabatch11.my.id/collection/delete?id_collection=' +
        oneCollection.id,
      headers: {Authorization: `bearer ${token}`},
    };
    axios(config)
      .then(function (response) {
        console.log(response.data);
        props.navigation.goBack();
        dispatch(actionGetAllCollection());
      })
      .catch(function (error) {
        alert(error.message);
      });
  };

  return (
    <View style={styles.container}>
      <StatusBar barStyle="light-content" />
      <ImageBackground
        style={styles.cover}
        source={{uri: oneCollection?.cover}}>
        <SafeAreaView style={styles.coverText}>
          {/* {isLoading ? (
            <View style={styles.success}>
              <Text style={styles.successText}>
                Collection succesfully created
              </Text>
            </View>
          ) : null} */}
          <TouchableOpacity
            onPress={() => {
              dispatch(actionGetAllCollection());
              props.navigation.goBack();
            }}
            style={styles.backBtn}>
            <Icon name="chevron-left" size={40} color="white" />
          </TouchableOpacity>
          <View style={{alignItems: 'center'}}>
            <Text
              style={{
                fontSize: 28,
                color: 'white',
                fontFamily: 'Montserrat-Bold',
              }}>
              {oneCollection?.title}
            </Text>
            <Text style={{color: 'white', fontFamily: 'Montserrat-Regular'}}>
              {oneCollection?.date?.slice(0, 10)}
            </Text>
            <Text
              style={{
                color: 'white',
                marginTop: 10,
                fontFamily: 'Montserrat-Regular',
              }}>
              {oneCollection?.collectionImages?.length / 2} Photos
            </Text>
          </View>
          <TouchableOpacity
            onPress={() => setShow(!show)}
            style={styles.delete}>
            <Icon name="dots-vertical" size={30} color="white" />
          </TouchableOpacity>
          {show ? (
            <View style={styles.deleteWrapper}>
              <TouchableOpacity
                onPress={() => {
                  setLoading(true);
                  delCollection();
                }}
                style={styles.deleteBtn}>
                <Icon name="trash-can-outline" size={20} />
                <Text style={{fontFamily: 'Montserrat-Regular'}}>Delete</Text>
              </TouchableOpacity>
            </View>
          ) : null}
        </SafeAreaView>
      </ImageBackground>
      <View style={styles.content}>
        <View style={{marginBottom: 20}}>
          <View
            style={{
              alignItems: 'center',
              justifyContent: 'center',
              marginVertical: 15,
            }}>
            <Text
              style={{
                fontSize: 20,
                fontFamily: 'Montserrat-Bold',
              }}>
              {item.title}
            </Text>
            <Text style={{fontFamily: 'Montserrat-Regular'}}>
              {item.date.slice(0, 10)}
            </Text>
          </View>
          <View style={{flexDirection: 'row', justifyContent: 'space-evenly'}}>
            <Text style={{fontFamily: 'Montserrat-Regular'}}>
              {item.collectionImages.length / 2} Photos
            </Text>
            {/* <Text style={{fontFamily: 'Montserrat-Regular'}}>54 Views</Text> */}
            <Text style={{fontFamily: 'Montserrat-Regular'}}>
              {item.totalDownload} Download
            </Text>
          </View>
        </View>
        <View>
          <TouchableOpacity
            onPress={() =>
              props.navigation.navigate('EditCollection', {item: item})
            }
            style={styles.items}>
            <Text style={{fontFamily: 'Montserrat-Regular'}}>
              Edit Collection
            </Text>
            <Icon name="chevron-right" size={30} />
          </TouchableOpacity>
          <TouchableOpacity
            onPress={() => props.navigation.navigate('ChangeTheme')}
            style={styles.items}>
            <Text style={{fontFamily: 'Montserrat-Regular'}}>Theme</Text>
            <Icon name="chevron-right" size={30} />
          </TouchableOpacity>
          <TouchableOpacity
            onPress={() => props.navigation.navigate('Privacy')}
            style={styles.items}>
            <Text style={{fontFamily: 'Montserrat-Regular'}}>Privacy</Text>
            <Icon name="chevron-right" size={30} />
          </TouchableOpacity>
          <TouchableOpacity
            onPress={() => props.navigation.navigate('Download')}
            style={styles.items}>
            <Text style={{fontFamily: 'Montserrat-Regular'}}>Download</Text>
            <Icon name="chevron-right" size={30} />
          </TouchableOpacity>
        </View>
        <View style={styles.viewShare}>
          <TouchableOpacity
            onPress={() => {
              if (oneCollection.theme == '1') {
                props.navigation.navigate('Classic', {item: item});
              } else if (oneCollection.theme == '2') {
                props.navigation.navigate('Minimalist', {item: item});
              } else {
                props.navigation.navigate('DarkMode', {item: item});
              }
            }}
            style={styles.view}>
            <Icon name="eye-outline" size={22} />
            <Text style={{marginLeft: 10, fontFamily: 'Montserrat-Regular'}}>
              View
            </Text>
          </TouchableOpacity>
        </View>
      </View>
      <Loading isVisible={loading} />
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: 'white',
  },
  cover: {
    flex: 0.5,
  },
  coverText: {
    justifyContent: 'center',
    alignItems: 'center',
    flex: 1,
    backgroundColor: 'rgba(1,1,1,0.4)',
  },
  backBtn: {
    position: 'absolute',
    left: 0,
    top:50
  },
  delete: {
    position: 'absolute',
    right: 0,
    top:50
  },
  deleteWrapper: {
    backgroundColor: 'white',
    borderRadius: 5,
    position: 'absolute',
    right: 10,
    bottom: 90,
  },
  deleteBtn: {
    flexDirection: 'row',
    alignItems: 'center',
    width: 80,
    justifyContent: 'space-around',
    paddingVertical: 8,
    paddingHorizontal: 4,
  },
  content: {
    flex: 1,
    paddingHorizontal: 30,
  },
  success: {
    backgroundColor: colors.info,
    borderRadius: 14,
    position: 'absolute',
    top: 50,
  },
  successText: {
    color: 'white',
    paddingHorizontal: 20,
    paddingVertical: 10,
    fontFamily: 'Montserrat-Regular',
  },
  items: {
    flexDirection: 'row',
    justifyContent: 'space-between',
    alignItems: 'center',
    width: '100%',
    backgroundColor: '#F8F8F8',
    padding: 10,
    marginVertical: 6,
    borderRadius: 4,
  },
  viewShare: {
    flexDirection: 'row',
    justifyContent: 'center',
    flex: 1,
    alignItems: 'center',
  },
  view: {
    flexDirection: 'row',
    alignItems: 'center',
    backgroundColor: colors.vogue,
    paddingHorizontal: 25,
    paddingVertical: 10,
    marginHorizontal: 10,
  },
  share: {
    flexDirection: 'row',
    alignItems: 'center',
    backgroundColor: colors.secondary,
    paddingHorizontal: 25,
    paddingVertical: 10,
    marginHorizontal: 10,
  },
});
