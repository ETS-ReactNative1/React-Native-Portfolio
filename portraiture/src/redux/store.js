import {createStore, applyMiddleware} from 'redux';
import createSagaMiddleware from 'redux-saga';
import combineReducers from './reducers/combiner'
import sagas from './sagas/Sagas'

const sagaMiddleware = createSagaMiddleware();
const store = createStore(combineReducers,applyMiddleware(sagaMiddleware));

sagaMiddleware.run(sagas);

export default store;