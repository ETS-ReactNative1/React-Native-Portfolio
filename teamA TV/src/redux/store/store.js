import ReduxThunk from 'redux-thunk';
import {createStore, applyMiddleware} from 'redux';
import rootReducer from '../reducer/combineReducer'


export let store = createStore(rootReducer,applyMiddleware(ReduxThunk))
