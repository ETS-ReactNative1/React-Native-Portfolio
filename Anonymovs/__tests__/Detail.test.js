import 'react-native';
import React from 'react';
import Detail from '../src/pages/HomeScreen/Detail';
import renderer from 'react-test-renderer';
import {Provider} from 'react-redux';
import store from '../src/redux/store';
jest.useFakeTimers();

test('Detail SnapShoot', () => {
  const snap = renderer
    .create(
      <Provider store={store}>
        <Detail />
      </Provider>,
    )
    .toJSON();

  expect(snap).toMatchSnapshot();
});
it(`renders correctly`, () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Detail />
      </Provider>,
    )
    .toJSON();
  expect(tree.children.length).toBe(1);
});
