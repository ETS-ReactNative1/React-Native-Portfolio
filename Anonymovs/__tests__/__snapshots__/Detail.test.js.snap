// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detail SnapShoot 1`] = `
<RCTScrollView
  showsVerticalScrollIndicator={false}
  style={
    Object {
      "backgroundColor": "#191926",
      "flex": 1,
    }
  }
>
  <View>
    <View
      accessibilityIgnoresInvertColors={true}
      style={
        Object {
          "resizeMode": "cover",
          "width": 750,
        }
      }
    >
      <Image
        source={
          Object {
            "uri": "https://image.tmdb.org/t/p/w500undefined",
          }
        }
        style={
          Array [
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
            Object {
              "height": undefined,
              "width": 750,
            },
            undefined,
          ]
        }
      />
      <BVLinearGradient
        colors={
          Array [
            0,
            4279834918,
          ]
        }
        endPoint={
          Object {
            "x": 0,
            "y": 1,
          }
        }
        locations={null}
        startPoint={
          Object {
            "x": 0,
            "y": 0,
          }
        }
        style={
          Object {
            "alignItems": "center",
            "height": 733.5,
            "justifyContent": "flex-end",
            "width": "100%",
          }
        }
      />
      <View
        style={
          Object {
            "elevation": 20,
            "shadowColor": "#fff",
            "shadowOffset": Object {
              "height": 0,
              "width": 0,
            },
            "shadowOpacity": 0.39,
            "shadowRadius": 8.3,
          }
        }
      >
        <Image
          resizeMode="contain"
          source={
            Object {
              "uri": "https://image.tmdb.org/t/p/w500undefined",
            }
          }
          style={
            Object {
              "bottom": 133.5,
              "height": 400,
              "left": 37.5,
              "position": "absolute",
              "width": 300,
            }
          }
        />
      </View>
    </View>
    <View
      style={
        Object {
          "marginHorizontal": 37.5,
          "top": -93.5,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "space-between",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#FFFFFF",
              "fontSize": 19,
              "fontWeight": "600",
            }
          }
        />
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "#FF3365",
                  "fontSize": 46.5,
                },
                undefined,
                Object {
                  "fontFamily": "Material Design Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            󰋕
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
          }
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "gold",
                "fontSize": 20,
              },
              undefined,
              Object {
                "fontFamily": "Material Design Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          󰓎
        </Text>
        <Text
          style={
            Object {
              "color": "#FFFFFF",
              "marginLeft": 10,
            }
          }
        >
          NaN
        </Text>
      </View>
      <Text
        style={
          Object {
            "color": "#999999",
            "marginTop": 26.5,
            "textAlign": "justify",
          }
        }
      />
      <View
        style={
          Object {
            "marginVertical": 26.5,
          }
        }
      >
        <View
          style={
            Object {
              "height": 220,
              "width": undefined,
            }
          }
        >
          <View
            style={
              Array [
                Object {
                  "flex": 1,
                  "overflow": "hidden",
                },
                undefined,
              ]
            }
          >
            <RNCWebView
              allowsFullscreenVideo={true}
              allowsInlineMediaPlayback={true}
              bounces={false}
              cacheEnabled={true}
              injectedJavaScriptBeforeContentLoadedForMainFrameOnly={true}
              injectedJavaScriptForMainFrameOnly={true}
              javaScriptEnabled={true}
              mediaPlaybackRequiresUserAction={false}
              messagingEnabled={true}
              onContentProcessDidTerminate={[Function]}
              onHttpError={[Function]}
              onLoadingError={[Function]}
              onLoadingFinish={[Function]}
              onLoadingProgress={[Function]}
              onLoadingStart={[Function]}
              onMessage={[Function]}
              onShouldStartLoadWithRequest={[Function]}
              source={
                Object {
                  "uri": "https://lonelycpp.github.io/react-native-youtube-iframe/iframe.html?data=%7B%22rel_s%22:0,%22loop_s%22:0,%22videoId_s%22:%22%22,%22controls_s%22:1,%22contentScale_s%22:1,%22cc_lang_pref_s%22:%22%22,%22allowWebViewZoom%22:false,%22modestbranding_s%22:0,%22preventFullScreen_s%22:1,%22showClosedCaptions_s%22:0%7D",
                }
              }
              style={
                Array [
                  Object {
                    "flex": 1,
                    "overflow": "hidden",
                  },
                  Object {
                    "backgroundColor": "#ffffff",
                  },
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                    },
                    undefined,
                  ],
                ]
              }
              useSharedProcessPool={true}
              userAgent=""
            />
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "marginBottom": 40,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#FFFFFF",
              "fontWeight": "500",
              "textAlign": "center",
            }
          }
        >
          Rate this Movie
        </Text>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "transparent",
                "flexDirection": "column",
                "justifyContent": "center",
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "fontWeight": "bold",
                  "margin": 10,
                },
                Object {
                  "color": "rgba(230, 196, 46, 1)",
                  "fontSize": 25,
                },
              ]
            }
          >
            So So
          </Text>
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "justifyContent": "center",
                },
              ]
            }
          >
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../react-native-ratings/dist/images/airbnb-star.png",
                  }
                }
                style={
                  Object {
                    "height": 20,
                    "margin": 3,
                    "tintColor": "#f1c40f",
                    "transform": Array [
                      Object {
                        "scale": 1,
                      },
                    ],
                    "width": 20,
                  }
                }
              />
            </View>
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../react-native-ratings/dist/images/airbnb-star.png",
                  }
                }
                style={
                  Object {
                    "height": 20,
                    "margin": 3,
                    "tintColor": "#f1c40f",
                    "transform": Array [
                      Object {
                        "scale": 1,
                      },
                    ],
                    "width": 20,
                  }
                }
              />
            </View>
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../react-native-ratings/dist/images/airbnb-star.png",
                  }
                }
                style={
                  Object {
                    "height": 20,
                    "margin": 3,
                    "tintColor": "#f1c40f",
                    "transform": Array [
                      Object {
                        "scale": 1,
                      },
                    ],
                    "width": 20,
                  }
                }
              />
            </View>
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../react-native-ratings/dist/images/airbnb-star.png",
                  }
                }
                style={
                  Object {
                    "height": 20,
                    "margin": 3,
                    "tintColor": "#f1c40f",
                    "transform": Array [
                      Object {
                        "scale": 1,
                      },
                    ],
                    "width": 20,
                  }
                }
              />
            </View>
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../react-native-ratings/dist/images/airbnb-star.png",
                  }
                }
                style={
                  Object {
                    "height": 20,
                    "margin": 3,
                    "tintColor": "#f1c40f",
                    "transform": Array [
                      Object {
                        "scale": 1,
                      },
                    ],
                    "width": 20,
                  }
                }
              />
            </View>
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../react-native-ratings/dist/images/airbnb-star.png",
                  }
                }
                style={
                  Object {
                    "height": 20,
                    "margin": 3,
                    "tintColor": "#BDC3C7",
                    "transform": Array [
                      Object {
                        "scale": 1,
                      },
                    ],
                    "width": 20,
                  }
                }
              />
            </View>
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../react-native-ratings/dist/images/airbnb-star.png",
                  }
                }
                style={
                  Object {
                    "height": 20,
                    "margin": 3,
                    "tintColor": "#BDC3C7",
                    "transform": Array [
                      Object {
                        "scale": 1,
                      },
                    ],
                    "width": 20,
                  }
                }
              />
            </View>
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../react-native-ratings/dist/images/airbnb-star.png",
                  }
                }
                style={
                  Object {
                    "height": 20,
                    "margin": 3,
                    "tintColor": "#BDC3C7",
                    "transform": Array [
                      Object {
                        "scale": 1,
                      },
                    ],
                    "width": 20,
                  }
                }
              />
            </View>
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../react-native-ratings/dist/images/airbnb-star.png",
                  }
                }
                style={
                  Object {
                    "height": 20,
                    "margin": 3,
                    "tintColor": "#BDC3C7",
                    "transform": Array [
                      Object {
                        "scale": 1,
                      },
                    ],
                    "width": 20,
                  }
                }
              />
            </View>
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../react-native-ratings/dist/images/airbnb-star.png",
                  }
                }
                style={
                  Object {
                    "height": 20,
                    "margin": 3,
                    "tintColor": "#BDC3C7",
                    "transform": Array [
                      Object {
                        "scale": 1,
                      },
                    ],
                    "width": 20,
                  }
                }
              />
            </View>
          </View>
        </View>
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#FF3365",
              "borderRadius": 15,
              "justifyContent": "center",
              "marginTop": 26.5,
              "opacity": 1,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#FFFFFF",
                "paddingVertical": 13.5,
              }
            }
          >
            Submit
          </Text>
        </View>
      </View>
      <View />
    </View>
  </View>
</RCTScrollView>
`;
